app/views/merchant_dashboards/index.html.erb

```diff
@@ -9,7 +9,7 @@
 <h4>Favorite Customers</h4>
 <ol>
 <% @merchant.favorite_customers.each do |customer| %>
-  <li><%= customer.first_name %> <%= customer.last_name %> - <%= customer.successful_transactions(@merchant.id) %> purchases</li>
+  <li><%= customer.first_name %> <%= customer.last_name %> - <%= customer.successful_transactions %> purchases</li>
```

Comment:
just flagging this because we'll probably run into a merge conflict here

Reply:
as long as you remember which one you want.  lol